docs/references/expo/expo-web-support.mdx

```diff
@@ -0,0 +1,93 @@
+---
+description: Learn how to add web support to your Expo project with Clerk.
+---
+
+# Expo web support
+
+Expo provides a way to [develop web applications](https://docs.expo.dev/workflow/web/) using the same codebase as your iOS and Android apps. Clerk supports web applications built with Expo, allowing you to use Clerk's prebuilt components to add authentication and user management to your web app.
+
+## Creating a new project with web support
+
+If you're starting from scratch, you can follow the [Expo quickstart](https://clerk.com/docs/quickstarts/expo), which directs you to create custom flows that can be used on all platforms out of the box.
+
+## Adding web support to an existing project
+
+If you already have an Expo project and want to add web support, you must first ensure that your Clerk custom flows do not use any native-specific code. If you have any native-specific code, in your custom flows, you will need to adjust it to work on the web, otherwise you will have to create a new custom flow specifically for the web or use the [prebuilt UI components](http://localhost:3000/docs/components/overview).
+
+## Use prebuilt components on web
+
+You can use Clerk's prebuilt components to add authentication and user management to your web app, make sure to follow the [Expo quickstart](https://clerk.com/docs/quickstarts/expo) if you start from scratch.
```

Comment:
Since this was mentioned above, we probably don't need another direction to the quickstart. 

I see below we dive into how to import the components, but do we need to callout that we need to do platform checks and/or create the file with the `.web` extention? 

If that's the case, maybe we can mention that in this section?

Reply:
Thank you Dom, fixed!